media-gfx/blender/blender-2.72b-r4.ebuild

```diff
@@ -246,8 +247,7 @@ src_install() {
 		dodoc -r "${CMAKE_USE_DIR}"/doc/doxygen/html/*
```

Comment:
1. delete the `src_test() { :; }`
2. Add `RESTRICT="test"` after `REQUIRED_USE`
3. Add a comment why